6_nodejs/week1/avg.js

```diff
@@ -0,0 +1,14 @@
+
+const args = require('minimist')(process.argv.slice(2));
```

Comment:
What if the user enters 2 or less arguments?
In that case it will be good to show them a error message which guides them into the correct usage of the code.

Reply:
Hey
Sorry for the late reply
What I could think of is using the trim() function on the argument to remove the spaces, and then doing a boolean check (!true).
If the argument is only whitespace, trim() will turn it into an empty string and as a result it becomes false in the boolean check. 
Hope I understood your question correctly - or otherwise let me know :)

Reply:
Oh okey, I think I got it. Thanks for your help!